stdlib/public/runtime/Casting.cpp

```diff
@@ -1406,7 +1406,6 @@ swift::swift_dynamicCastMetatypeUnconditional(const Metadata *sourceType,
   }
 }
 
-#if SWIFT_OBJC_INTEROP
 /// Do a dynamic cast to the target class.
 static void *_dynamicCastUnknownClass(void *object,
```

Comment:
This entry point should not be needed on the native runtime.  There are no unknown objects in the native runtime, all objects are Swift objects.